src/rxjs/rxjs.md

```diff
@@ -75,13 +75,13 @@ be a useful reference for understanding how the tutorial works:
 
 - Open Developer Tools and stack CodePen vertically:
 
-  <img src="./static/img/codepen-stack.png" width="50%" alt="CodePen stacked" title="CodePen stacked" style="border: solid 1px #808080" />
+  <img src="./static/img/codepen-stack.png" width="50%" alt="CodePen stacked" alt="CodePen stacked" style="border: solid 1px #808080" />
 
 - Save your CodePen
 - If you can’t figure out what is breaking, use an [online diff checker](https://www.diffchecker.com/) to
   compare your code with the solution.
 
-  <img src="./static/img/diff-checker.png" width="50%" alt="Diff Checker" title="DiffChecker" style="border: solid 1px #808080" />
+  <img src="./static/img/diff-checker.png" width="50%" alt="Diff Checker" alt="DiffChecker" style="border: solid 1px #808080" />
```

Comment:
`alt` is specified twice here (and elsewhere).

Reply:
👀

Reply:
One more thing: some of the images are rendered with the Markdown syntax and have titles specified, so please search for `![` and remove the titles.